front-end/src/ui-components/AddressDropdown.tsx

```diff
@@ -51,7 +51,7 @@ const AddressDropdown = ({ accounts, className, defaultAddress, filterAccounts,
 
 	return <Dropdown
 		className={className}
-		pointing='bottom'
+		pointing
```

Comment:
Just to be sure (I was wrong with the bottom obviously), not entirely sure what the default is, the doc doesn't say..
```suggestion
		pointing='top'
```